### ISSUE TITLE
Please Provide Proper Documentation for Components uses

### ISSUE THREAD

**Issue by VimLeSai** [carbon-design-system/carbon#6622]

**Comment by joshblack** [comment]
Hi there @VimLeSai! 👋  We have a storybook available for teams using React over here: https://react.carbondesignsystem.com/ along with the usage docs on the site itself.

We're actively working on improving these docs, as well! Is there anything you'd like to see in these updates docs that isn't currently available? Would love to help make it happen 😄

**Comment by joshblack** [comment]
Going to close this for now, feel free to comment with any suggestions that you might have! Would love to hear them 😄
